cdk/lib/__snapshots__/cdk-stack.test.ts.snap

```diff
@@ -321,7 +321,6 @@ Object {
     },
     "tagjanitorlambdatagjanitorlambdarate7days0B5F68379": Object {
       "Properties": Object {
-        "Description": "Run tag-janitor every 7 days",
```

Comment:
This has been dropped as [the new pattern doesn't take a description](https://github.com/guardian/cdk/blob/main/src/patterns/scheduled-lambda.ts#L8-L11). I think this is pretty clear from the `rate(7 days)` and the `Targets` block anyway.